Cargo.toml

```diff
@@ -12,19 +12,22 @@ repository = "https://github.com/fluffysquirrels/mqtt-async-client-rs"
 bytes = "0.4.0"
 futures-core = "0.3.1"
 futures-util = "0.3.1"
+http = "0.2.4"
 log = "0.4.8"
 maplit = "1.0.2"
 mqttrs = "0.2.0"
 rustls = { version = "0.19.0", optional = true }
 tokio = { version = "1.2.0", features = ["io-util", "macros", "net", "rt", "rt-multi-thread", "sync", "time"] }
 tokio-rustls = { version = "0.22.0", optional = true }
+tokio-tungstenite = { version = "0.14", optional = true, features = ["rustls-tls"] }
 
 [dev-dependencies]
 env_logger = "0.7.1"
 structopt = "0.3.5"
 webpki-roots = "0.18.0"
 
 [features]
-default = ["tls"]
+default = ["tls", "websocket"]
```

Comment:
I think we should not include the "websocket" feature by default.

Also, please document the "websocket" feature in `${REPO}/src/lib.rs` as the "tls" feature is.

Reply:
Done!